dev-libs/serdisplib/files/disable-static-build.patch

```diff
@@ -0,0 +1,95 @@
+--- a/server/Makefile.in	2018-05-15 21:23:18.370830011 +0200
++++ b/server/Makefile.in	2018-05-15 21:24:27.264829748 +0200
+@@ -83,14 +83,14 @@
+ .c.o:
+ 	$(CC) $(CFLAGS) $(DEFINES)  -c $(INCLUDES) $<
+ 
+-serdispd: $(LIB_DIR)/$(LIB_STATIC) $(SDISPDOBJECTS)
+-	$(CC) -o $@ $(SDISPDOBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
```

Comment:
Still, isn't EXTRA_LIBS_STATIC empty at this point?

Reply:
Yes. You are right. Dropped it.